src/main/scripts/src/app/workspace/workspace-chooser/workspace-chooser.component.ts

```diff
@@ -107,59 +111,57 @@ export class WorkspaceChooserComponent implements OnInit {
     this.activeProject = undefined;
   }
 
-  public onProjectSelected(project: Project, index: number) {
+  public onProjectSelected(project: Project, index: number): void {
     this.activeOrganization.projects.forEach((oneProject: Project) => oneProject.active = false);
     project.active = true;
     project.index = index;
     this.activeProject = project;
   }
 
-  public onScrollOrganizations(toRight?: boolean) {
-    if (toRight) {
-      this.companiesEl.scrollToLeft(this.companiesEl.elementRef.nativeElement.scrollLeft + squareSize);
-    } else {
-      this.companiesEl.scrollToLeft(this.companiesEl.elementRef.nativeElement.scrollLeft - squareSize);
-    }
+  public onScrollOrganizations(direction: number): void {
+    this.companiesElement.scrollToLeft(this.companiesElement['elementRef'].element.scrollLeft + squareSize * direction);
   }
 
-  public onScrollProjects(toRight?: boolean) {
+  public onScrollProjects(toRight?: boolean): void {
     if (toRight) {
-      this.projectsEl.scrollToLeft(this.projectsEl.elementRef.nativeElement.scrollLeft + squareSize);
+      this.projectsElement.scrollToLeft(this.projectsElement['elementRef'].element.scrollLeft + squareSize);
```

Comment:
I am not able to compile the application on my computer because of this change. When I change it back to `nativeElement`, it seems to work.